doc/howto/network_bridge_firewalld.md

```diff
@@ -73,17 +73,19 @@ Depending on your use case, you might need more advanced rules and the example c
 ### UFW: Add rules for the bridge
 
 If UFW has a rule to drop all unrecognized traffic, it blocks the traffic to and from the LXD bridge.
-In this case, you must add rules to allow traffic to and from the bridge.
+In this case, you must add rules to allow traffic to and from the bridge, as well allow traffic as forwarded to it.
```

Comment:
"as well as allowing traffic that is forwarded to it" reads better in my opinion.

Reply:
I agree, fixed